source/_integrations/purpleair.markdown

```diff
@@ -17,3 +17,9 @@ ha_config_flow: true
 [PurpleAir](https://www2.purpleair.com/) makes sensors that measure hyper-local air quality data and share it with the public.
 
 {% include integrations/config_flow.md %}
+
+{% include integrations/option_flow.md %}
```

Comment:
The options flow does not provide a header.

In general, when referring to the options flow, including a list of options using a basic configuration block is helpful.

Reply:
> The options flow does not provide a header.

It doesn't? Isn't that what this is? https://github.com/home-assistant/home-assistant.io/blob/7d82e68de0655f3cb794e6727c1b388b22c46799/source/_includes/integrations/option_flow.md?plain=1#L3

Reply:
Oh darn! You are right!

Sorry about that 👍